src/cmd/support_archive/objects_query.go

```diff
@@ -0,0 +1,101 @@
+package support_archive
```

Comment:
would rename this file to resource_query.go e.g

Reply:
consequently all other `objects*.*` files have to be renamed too